src/adminApp/user.js

```diff
@@ -184,13 +184,22 @@ const ConceptSyncAttributeShow = ({ subjectType, syncAttributeName, ...props })
   const syncSettings = get(props.record, ["syncSettings", subjectType.name], {});
   const conceptUUID = get(syncSettings, [syncAttributeName]);
   if (isEmpty(conceptUUID)) return null;
+
+  const [valueMap, setValueMap] = useState(new Map());
+  useEffect(() => {
+    const newValMap = new Map();
+    ConceptService.getAnswerConcepts(conceptUUID).then(content =>
+      content.forEach(val => newValMap.set(val.id, val.name))
+    );
+    setValueMap(newValMap);
+  }, []);
```

Comment:
1. Since you are depending on a props value, you should declare it in the dependencies - https://react.dev/reference/react/useEffect#examples-dependencies
2. valueMap is a generic name. Add something more meaningful. eg: conceptSyncAttributeValues